scripts/resources/dns-server.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: submariner-lighthouse
```

Comment:
This is already created by helm. We create it for lighthouse-agent and can reuse for dns server